impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -169,7 +171,7 @@ static void DrawGlyph(SkCanvas* canvas,
   sk_font.setHinting(SkFontHinting::kSlight);
   sk_font.setEmbolden(metrics.embolden);
 
-  auto glyph_color = has_color ? SK_ColorWHITE : SK_ColorBLACK;
+  auto glyph_color = has_color ? SK_ColorWHITE : SK_ColorRED;
```

Comment:
Shouldn't opaque white do the trick always? Why does Skia care what we interpret the single channel to be?

Reply:
I think the problem was that I set the colorspace to red too. This should stay as white/greyscale and then like you said - the backend can choose to interpret the one channel bitmap as either grey or red.